commands/Char.js

```diff
@@ -0,0 +1,56 @@
+import { MessageEmbed } from "@yuuto-project/discord.js";
+import Command from "./base/Command.js";
+// Adapted from: https://github.com/dunste123/hirobot/blob/master/valentines.json
+import chars from "../assets/chars.json";
+
+export default class Char extends Command {
+  constructor() {
+    super({
+      name: "char",
+      category: "info",
+      aliases: ["character", "characters"],
+      description: "Show information about a specific character.",
+      usage: "Run `char <character>` to see information about a character."
+    });
+  }
+
+  run(client, { channel }, args) {
+    if (!args.length) {
+      channel.send("Please provide a character.");
+      return;
+    }
+    const characterName = args[0];
+    const characterInfo = this.getCharacterInfo(characterName.toLowerCase());
+
+    if (!characterInfo) {
+      channel.send(
+        `Sorry, but I don't think ${characterName} is a character in Camp Buddy.`
+      );
+      return;
+    }
+
+    channel.send(this.getEmbed(characterInfo));
+  }
+
+  getCharacterInfo(nameToFind) {
+    return chars.find(info => {
+      return info.name.toLowerCase().includes(nameToFind);
+    });
```

Comment:
This looks nicer imo
```suggestion
    return chars.find(
        { name } => name.toLowerCase().includes(nameToFind)
    );
```

Reply:
I get an error with that
```
SyntaxError: Malformed arrow function parameter list
```

Reply:
oh right, it needs parenthesis `({ name }) => ...`